test/flycheck-test.el

```diff
@@ -3885,6 +3885,13 @@ Why not:
      '(5 9 warning "Redundant braces after class definition"
          :checker scala-scalastyle))))
 
+(flycheck-ert-def-checker-test scheme-chicken scheme nil
+  (skip-unless (funcall (flycheck-checker-get 'chicken-scheme 'predicate)))
```

Comment:
@manuel-uberti You'd temporarily add a hook to `scheme-mode-hook` that enables Geiser.  Take a look at the `does-not-check-with-no-byte-compile` test for Emacs Lisp to see how to add a hook for just for one test case.

The hook function should then do whatever is required to setup Geiser Mode.